requirements.txt

```diff
@@ -5,7 +5,10 @@ numpy>=1.15.2
 scipy>=1.1.0
 tqdm>=4.26.0
 FCD==1.1
-
+# FCD doesn't pin dependencies, so we have to
```

Comment:
The sentence is not fully complete: "so we have to ..."?

Not sure if this is on purpose; same in `setup.py`.

Reply:
Sorry, I meant that because FCD doesn't pin the dependencies, we have to [pin the dependencies] 😄 
```suggestion
# FCD doesn't so we have to pin the dependencies
```